tests/eclient/testdata/app_logs.txt

```diff
@@ -29,6 +30,7 @@ stdout 'ubuntu-http-server'
 wait
 
 eden pod delete eclient
+eden network delete n1
 
 test eden.app.test -test.v -timewait 10m - eclient
```

Comment:
Please add `test eden.network.test -test.v -timewait 10m - n1` after `test eden.app.test` to wait for network delete process.
Seems we also need something like
```
message 'Resetting of EVE'
eden eve reset
exec sleep 30
```
in the end of tests/eclient/profile.txt to not keep network from the previous test. Now I can see [network with defined parameters already exists](https://github.com/lf-edge/eden/actions/runs/4927301822/jobs/8804097337?pr=864#step:11:4306) complain.

Reply:
Won't deleting and waiting for deletion of this network solve the problem? :)

Reply:
The error indicates that we have network from the previous test (profile) and we cannot create another network with the same subnet.

Reply:
If we delete network in profile test and run
`test eden.network.test -test.v -timewait 10m -n1` won't it solve the problem?

Reply:
Yes, if we will explicitly create and delete network there, it should solve the problem.